client/src/components/Youtube/Youtube.jsx

```diff
@@ -0,0 +1,20 @@
+import { YoutubeBox } from "./YoutubeStyle";
+
+function Youtube({ songName, artistName, youtubeUrl }) {
+  return (
+    <div>
+      <YoutubeBox>
+        <iframe
+          width="560"
```

Comment:
I think the numbers should be dynamic

Reply:
no its should be the same for all the videos i check this with a few videos